src/color-slider/color-slider.js

```diff
@@ -63,7 +63,10 @@ const Self = class ColorSlider extends NudeElement {
 		if (["min", "max", "step", "value", "defaultValue"].includes(name)) {
 			prop.applyChange(this._el.slider, change);
 
-			let spinnerValue = this.tooltip === "progress" ? +(this.progress * 100).toPrecision(4) : this.value;
+			let spinnerValue = change.value;
```

Comment:
Nit: I would call it just `value`, not `spinnerValue`.

Reply:
Fixed. Thanks!